crbytes/common_prefix_amd64.s

```diff
@@ -0,0 +1,166 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in licenses/BSD-golang.txt.
+
+// This code is based on compare_amd64.s from Go 1.12.5.
+
+TEXT ·CommonPrefix(SB),$0-56
```

Comment:
If anyone actually wants to look at this, here's the diff against `bytes.Compare`: https://editor.mergely.com/V6Bvcmbr